CONTRIBUTING.md

```diff
@@ -14,9 +14,9 @@ Here are a few things to keep in mind when making changes to DefectDojo.
 
 ## Modifying DefectDojo and Testing
 
-Please use [these test scripts](./tests) to test your changes. These are the exact scripts we run in our [Travis Build](https://travis-ci.org/OWASP/django-DefectDojo).
+Please use [these test scripts](./tests) to test your changes. These are the scripts we run in our integration tests.
```

Comment:
should we point them to the docker page which mentions both unit tests and integration tests?

Reply:
You mean this? DOCKER.md#run-the-tests-with-docker